docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -75,6 +75,14 @@ This reference details available values for the `teleport-cluster` chart.
 
 `proxyListenerMode` controls proxy TLS routing used by Teleport. Possible values are `multiplex`.
 
+## `sessionRecording`
+
+| Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
+| - | - | - | - | - |
+| `string` | `node` | no | `auth_service.session_recording` | ❌ |
+
+`sessionRecording` controls session recording mode. Possible values are `node`, `node-sync`, `proxy`, `proxy-sync`, `off`.
```

Comment:
If we're already referring the user to the configuration reference docs, I think it would be more informative to say that the value of `sessionRecording` is passed unchanged to the `session_recording` config field than to list the possible values without explaining them.

Reply:
I'm copying the format other configuration values do similarly in this document.  For example `authenticationType` and `proxyListenerMode` above.